sdk/registry.go

```diff
@@ -1,11 +1,26 @@
 package sdk
 
 import (
+	"regexp"
+	"runtime"
 	"sync"
 
 	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
 )
 
+// /path/.../global-hooks/a/b/c/Hook-name.go
+// $1 - Hook path
+// $3 - Hook name
+var globalRe = regexp.MustCompile(`/global-hooks/(([^/]+/)*([^/]+))$`)
```

Comment:
```suggestion
var globalRe = regexp.MustCompile(`/global[\/\-]hooks/(([^/]+/)*([^/]+))$`)
```

There are 2 layouts:

1. All hooks in `/global-hooks`
2. Hooks in `/global/hooks` and OpenAPI schemas in `/global/openapi` — there was a plan to use it.